### PR TITLE
[stable/ambassador] Prometheus exporter resource configuration support

### PR DESCRIPTION
**Pull request by ysaakpr**
#### What this PR does / why we need it:
There is no way to configure the prometheus exporter resource in the current chart. This will give the support for adding resource requests and limits to the prometheus exporter as well

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/ambassador/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.70.1
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 2.6.2
+version: 2.6.3
```
Comment (Flydiverny) [inline stable/ambassador/Chart.yaml]: Added functionallity -> minor :) 2.7.0

**Comment by ysaakpr** [comment]
@Flydiverny review changes are updated

**Pull request merged**